packages/paste-core/components/modal/__tests__/customization.spec.tsx

```diff
@@ -54,11 +54,6 @@ describe('Modal Customization', () => {
 
       const modalFooterActions = modalFooter.firstChild as HTMLElement;
       expect(modalFooterActions.getAttribute('data-paste-element')).toEqual('MODAL_FOOTER_ACTIONS');
-
-      const modalFooterActionItemOne = modalFooterActions.firstChild as HTMLElement;
-      const modalFooterActionItemTwo = modalFooterActions.lastChild as HTMLElement;
-      expect(modalFooterActionItemOne.getAttribute('data-paste-element')).toEqual('MODAL_FOOTER_ACTIONS_ITEM');
-      expect(modalFooterActionItemTwo.getAttribute('data-paste-element')).toEqual('MODAL_FOOTER_ACTIONS_ITEM');
```

Comment:
issue: Unfortunately, the removal of this customization element target is a breaking change.